src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,22 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
+
+    List<Car> getAll();
+
+    Car update(Car car);
+
+    boolean delete(Long id);
+
+    List<Car> getAllByDriver(Long driverId);
+
+    List<Driver> getDriversForCar(Long id);
```

Comment:
you don't need to add this method on interface level, you can use it inside, just make it private